KWZP2019/KWZP2019/Production/ProductionProcess.cs

```diff
@@ -21,22 +22,64 @@ public ProductionProcess(RoofingCompanyEntities db)
 
         private void ProductionProcess_Load(object sender, EventArgs e)
         {
+            dTPickerProductionActualFilterUpper.Value = DateTime.Today;
+            dTPickerProductionActualFilterLower.Value = DateTime.Today.AddDays(-30);
```

Comment:
co jesli miesiac ma 28 albo 31 dni? :)

Reply:
W tym przypadku miałem na myśli domyślne ustawienie filtru na 30 dni wstecz, bez znaczenia jaki jest/był miesiąc

Reply:
w porządku, chciałem żeby to było swiadome działanie i jak widze- jest